client/signup/steps/site-or-domain/index.jsx

```diff
@@ -67,12 +70,21 @@ export default class SiteOrDomain extends Component {
 		return (
 			<div className="site-or-domain__choices">
 				{ this.getChoices().map( ( choice ) => (
-					<Card className="site-or-domain__choice" key={ choice.type }>
-						<a href="#" onClick={ ( event ) => this.handleClickChoice( event, choice.type ) }>
-							{ choice.image }
-							<h2>{ choice.label }</h2>
-						</a>
-					</Card>
+					<div className="site-or-domain__choice" key={ choice.type }>
```

Comment:
I'd either make the whole boxes clickable, or just the button. It's a bit confusing that you can click the image/button but not outside those two.